hazelcast/src/main/java/com/hazelcast/config/HotRestartPersistenceConfig.java

```diff
@@ -46,8 +46,14 @@
      */
     public static final int DEFAULT_DATA_LOAD_TIMEOUT = 15 * 60;
 
+    /**
+     * Default number of HR stores per HZ instance
+     */
+    public static final int DEFAULT_STORE_COUNT = 1;
```

Comment:
thinking out-loud: we do need to expose the fact there are multiple stored used? Isn't this just an implementation detail? Isn't "parallelism" what's really important here?